### DISCUSSION TITLE
How to shrink the New Tab button

### DISCUSSION THREAD

**Discussion by DALEcz** [piroor/treestyletab#3456] [Q&A]
How to make the New Tab button area smaller? It is more than twice the size of a normal Tab (see the pic): 

![obrazek](https://github.com/piroor/treestyletab/assets/33961557/8b9c5b8d-e623-4df4-b985-7d9da5a32b82)

I was playing with
`.newtab-button {
  height: 10px;
}`
or
`.newtab-button-box {
  height: 15px;
 }`

 It only makes the functional / clickable area smaller, but the grey area is still the same size.

Thank you!

**Comment by irvinm** [comment]
Looks like there is some padding around the button you can get rid of to reduce the size.

![image](https://github.com/piroor/treestyletab/assets/979729/31091c36-19ba-4947-85c3-cc0acff53053)

**Reply by DALEcz** [reply]
Ah! Interesting, thanks - I will try to play with that.
